src/miniscript/satisfy.rs

```diff
@@ -500,11 +504,115 @@ impl_tuple_satisfier!(A, B, C, D, E, F);
 impl_tuple_satisfier!(A, B, C, D, E, F, G);
 impl_tuple_satisfier!(A, B, C, D, E, F, G, H);
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+/// Placeholder for some data in a [`WitnessTemplate`]
+pub enum Placeholder<Pk: MiniscriptKey> {
+    /// Public key and its size
+    Pubkey(Pk, usize),
+    /// Public key hash and its size
+    PubkeyHash(hash160::Hash, usize),
```

Comment:
In dc0b2804a8368bea9f5e6e4717ad31bc8933e090:

This comment should be updated to clarify that it's the size of the hashed pubkey, not the size of the hash